CMakeLists.txt

```diff
@@ -41,6 +42,13 @@ else()
 		PROPERTY SUFFIX ".${Python_SOABI}${CMAKE_SHARED_MODULE_SUFFIX}")
 endif()
 
-target_link_libraries(swig_wasserstein PUBLIC wasserstein OpenMP::OpenMP_CXX Python::Module Python::NumPy)
+target_link_libraries(
+	swig_wasserstein
+	PUBLIC
+	wasserstein
+	OpenMP::OpenMP_CXX
+	Python::Module
+	Python::NumPy
+	${PYTHON_LIBRARIES})
```

Comment:
No need to use `PYTHON_LIBRARIES` if you are using Python::Module, that include it. And this is mixing `PYTHON_LIBRARIES` (the old FindPythonLibs) with `Python_LIBRARIES` (the new FindPython).